Automattic/woocommerce-payments
# Fix WooPay component spacing

Fixes #9699

#### Changes proposed in this Pull Request









This PR fixes the WooPay component spacing and border issues on mobile.

| Existing email | Non existing email | Unchecked |
|-|-|-|
|  <img width="391" alt="Screenshot 2024-11-18 at 1 39 02 PM" src="https://github.com/user-attachments/assets/faee8109-5d4b-4312-9a6d-8f9c1888d57b"> | <img width="391" alt="Screenshot 2024-11-18 at 5 03 37 PM" src="https://github.com/user-attachments/assets/7d9a5c11-71ca-4b27-b26e-4f3dd7b0eafb"> | <img width="391" alt="Screenshot 2024-11-18 at 5 07 07 PM" src="https://github.com/user-attachments/assets/2cb93184-fb9f-4413-9612-531b68f68e2e"> |


#### Testing instructions





Test on desktop and mobile viewports.

* Enable WooPay
* Add products to the cart
* Access checkout page
* Type an existent WooPay address
* The WooPay `Save my info` should be shown
* The spacing and borders should be working as expected
* Type a non existent WooPay address
* The WooPay `Save my info` component should be hidden
* The spacing and borders should be the same as WooPay disabled

-------------------


**Post merge**
